### ISSUE TITLE
UnnecessaryStringOutput reports false positives when calling methods on a string literal

### ISSUE THREAD

**Issue by glittershark** [causes/haml-lint#71]
For example:
```haml
= 'user'.pluralize(@users.count)
```

**Comment by sds** [comment]
Thanks for the bug report, @glittershark!

Fixed in 5c63794
